tensorflow/contrib/memory_stats/python/kernel_tests/memory_stats_ops_test.py

```diff
@@ -32,7 +32,7 @@ class MemoryStatsOpsTest(test_util.TensorFlowTestCase):
   def testBytesLimit(self):
     # AllocatorStats.bytes_limit is set to zero for CPU allocators, so we skip
     # the check.
-    if not test.is_gpu_available():
+    if not test.is_gpu_available() or test_util.is_sycl_enabled():
```

Comment:
Removed in https://github.com/tensorflow/tensorflow/pull/9117/commits/73e405572a59ba8ab1d92dc3f7cddff5c9fb240c

The intention behind disabling the failing tests was to establish a regression baseline so that we can establish a smooth development process. We would enable more tests as we add the corresponding features, while being able to continually test that we haven't introduced regressions.